templates/core/core-pre-upgrade-job.yaml

```diff
@@ -0,0 +1,173 @@
+{{- if .Values.core.enableMigrateHelmHook }}
```

Comment:
Many attributes in the job are copied from the core pod, but are not needed, they should be removed?  IMO the job should be much simpler.

Reply:
Updated